internal/services/netapp/netapp_volume_resource.go

```diff
@@ -317,6 +318,36 @@ func resourceNetAppVolume() *pluginsdk.Resource {
 				},
 			},
 
+			"data_protection_backup_policy": {
```

Comment:
Because this isn't ForceNew, could we add another step to the test to update this block?

Reply:
good point, added the test and fixed a bug.